pkg/pushsync/pushsync.go

```diff
@@ -369,6 +371,12 @@ func (ps *PushSync) pushToClosest(ctx context.Context, ch swarm.Chunk, retryAllo
 			}
 			if err != nil {
 				logger.Debugf("could not push to peer %s: %v", peer, err)
+
```

Comment:
It may be that this change affected the TestReplicateBeforeReceipt test.

Reply:
So we discovered that the granularity of the time pkg is not enough such that the warmup check returns false in some cases. We fixed it by changing the warmup time to some negative number. Tests should pass in the new commit.